skypy/neutral_hydrogen/mass.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+from astropy import units
+
+
+r"""HI mass module.
+This module provides functions to calculate neutral Hydrogen masses for haloes.
+
+"""
+
+__all__ = [
+           'pr_halo_model',
+]
+
+
+def pr_halo_model(halo_mass, circular_velocity, cosmology,
+                  alpha=0.17,
+                  beta=-0.55,
+                  v_c0=np.exp(1.57) * (units.km / units.s),
+                  v_c1=np.exp(4.39) * (units.km / units.s),
+                  Y_p=0.24):
+    r"""Model of Padmanabhan & Refregier (2017), equation 1.
+
+    Halo neutral Hydrogen (HI 21-cm) masses as a function of halo mass,
+    halo circular velocity, and cosmology.
+
+    Parameters
+    ----------
+    halo_mass : (nhalos,) `~astropy.Quantity`
+        The masses of the underlying halos in units of solar mass.
+    circular_velocity : (nhalos,) `~astropy.Quantity`
+        The circular velocity for the halos in units of [km s-1].
+    cosmology : `~astropy.cosmology.Cosmology`
+        Cosmology object providing values of omega_baryon and omega_matter.
```

Comment:
This would be better moved to the end of the parameter list because it will be filled by the pipeline runner automatically.